lib/irc.js

```diff
@@ -1213,6 +1213,10 @@ Client.prototype.notice = function(target, text) {
     this._speak('NOTICE', target, text);
 };
 
+Client.prototype.setTopic = function(target, text) {
```

Comment:
Can you add a test for this?

Reply:
Can you point me to a test I can take as an example? I'm not sure where to put it.